File: ProjectPendragonBackend/Models/CombatSkills.cs
namespace ProjectPendragonBackend.Models
{
    public class CombatSkills
    {
        public Guid Id { get; set; }
        public int Battle { get; set; }
        public int Siege { get; set; }
        public int Horsemanship { get; set; }
        public int Sword { get; set; }
        public int Lance { get; set; }
        public int Spear { get; set; }
        public int GreatSpear { get; set; }
        public int Dagger { get; set; }
        public int SpearExpertise { get; set; }
        public int GreatWeapon { get; set; }
    }
}

File: ProjectPendragon/src/app/models/character/combat-skills.ts
export class CombatSkills {
    id: string;
    battle: number;
    siege: number;
    horsemanship: number;
    sword: number;
    lance: number;
    spear: number;
    greatSpear: number;
    dagger: number;
    spearExpertise: number;
    greatWeapon: number;

    constructor() {
        this.id = crypto.randomUUID();
        this.battle = 0;
        this.siege = 0;
        this.horsemanship = 10;
        this.sword = 10;
        this.lance = 10;
        this.spear = 10;
        this.greatSpear = 0;
        this.dagger = 5;
        this.spearExpertise = 15;
        this.greatWeapon = 0;
    }
}